hackathon2.R

```diff
@@ -0,0 +1,90 @@
+# GOAL of this research: ARE WE IMPROVING on SCREENING of MUAC and OEDEMA? 
+##Are wee expanding our screening coverage? From baseline vs. endline? By sex? By location?
+
+## Install & Load R packages 
+install.packages("gtsummary")
+install.packages("dplyr")
+
+library(dplyr)
+library("gtsummary")
+
+## Load functions in R directory 
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+# Read data 
+nut<-read.csv(file="https://raw.githubusercontent.com/OxfordIHTM/ihtm-hackathon-2023/main/data/nutrition_survey_data.csv")  
+nut
+
+##Subset data: Transform variables of interest into table nut1 
+nut1 <- nut %>% select(survey_round,county,sex,muac_screen,oedema_screen)
+
+View(nut1)
+
+##Clean data in subset: Change 99 to NA 
+###Clean data only after having made subset nut1. Code don't work if data is cleaned in nut and then subset. 
+nut1$muac_screen <- ifelse(nut1$muac_screen == 99, "NA", nut1$muac_screen)
+nut1$muac_screen [nut$muac_screen == 1] <- "Y"
+nut1$muac_screen [nut$muac_screen == 2] <- "N"
+
+nut1$oedema_screen <- ifelse(nut1$oedema_screen == 99, "NA", nut1$muac_screen)
+nut1$oedema_screen [nut$oedema_screen == 1] <- "Y"
+nut1$oedema_screen [nut$oedema_screen == 2] <- "N"
+
+nut1$sex <- ifelse(nut1$sex == 1, "Male", nut1$sex)
+nut1$sex <- ifelse(nut1$sex == 2, "Female", nut1$sex)
+
+View(nut1)
```

Comment:
As we have learned in our earlier exercise, View(nut) is really something that you don't want to encode in a script. If using RStudio, you can just easily press on the object you want to see or just make a call to the object on the console.